peoplehr/tests/test.bal

```diff
@@ -348,8 +355,8 @@ function testGetAppraisalDetailsByEmployeeID() returns error? {
 function testGetAppraisalDetailsByAppraisalID() returns error? {
     log:printInfo("Get appraisal details by appraisal ID");
     AppraisalDetailsByAppraisalIDRequest payload = {
-        EmployeeId: "123",
-        AppraisalId: 0
+        EmployeeId: "PW10",
+        AppraisalId: 1
```

Comment:
`AppraisalId ` why it is `1` not `2`? 
Shall we make it a configurable or a constant?